src/sql/workbench/contrib/notebook/browser/calloutDialog/imageCalloutDialog.ts

```diff
@@ -197,9 +197,17 @@ export class ImageCalloutDialog extends Modal {
 		this._register(styler.attachCheckboxStyler(this._imageEmbedCheckbox, this._themeService));
 	}
 
+	private readonly doubleQuotesRegex = /^\"(.*)\"$/;
```

Comment:
Just have this be a const outside the class - no reason to have it as a class member. And this could probably just go in the common or utils folder too so you're not duplicating the regex.